backend/web/ui/static/js/app.js

```diff
@@ -63,3 +62,123 @@ function user_login() {
         dataType: "json",
     });
 }
+
+function add_new_todo(todo_status_id) {
+    let form_id = "#form_" + todo_status_id
+    $(form_id).trigger("reset");
+    var modal_id = "#modal_" + todo_status_id
+    $(form_id).removeClass('was-validated')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/IrinaLokhvytska/owasp/pullRequest?prid=8125955)
```suggestion
    $(form_id).removeClass("was-validated")
```